cabal.project

```diff
@@ -90,3 +95,9 @@ constraints:
   plutus-core < 1.51,
   plutus-ledger-api < 1.51,
   plutus-tx < 1.51,
+
+source-repository-package
+  type: git
+  location: https://github.com/jasagredo/digest
+  tag: 329fc2a911878ffe47472751cb40aae20ab2c00a
+  --sha256: sha256-84f8dFee9EfWbQ5UTLZ9MrsZ3JVojNhzfTGmWof6wHU=
```

Comment:
This can be conditional to just be used on Windows, right? (Might make sense to add a comment why it is necessary)

Also, what is the plan to make this unnecessary? Release this to CHaP for now, and then try to upstream?

Reply:
Good call

Reply:
I think we can close this as the digest patch that removes crc32c for windows is released in CHaP.